src/app/blog/_components/archives/theme-switcher.tsx

```
@@ -0,0 +1,113 @@
+    const resolvedMode = mode === SYSTEM ? systemMode : mode;
+    const classList = document.documentElement.classList;
+    if (resolvedMode === DARK) classList.add(DARK);
+    else classList.remove(DARK);
+    document.documentElement.setAttribute("data-mode", mode);
+    restoreTransitions();
+  };
+  window.updateDOM();
+  media.addEventListener("change", window.updateDOM);
+};
+
+let updateDOM: () => void;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'updateDOM'. Consider to delete it or rename it.

'updateDOM' is defined here:


(lint/suspicious/noRedeclare)